crates/pipeline-manager/src/api/endpoints/cluster_healthz.rs

```diff
@@ -0,0 +1,47 @@
+use actix_web::{get, web::Data as WebData, HttpResponse, Responder};
+use chrono::Utc;
+
+use crate::{
+    api::main::ServerState,
+    cluster_health::{HealthStatus, ServiceStatus},
+};
+
+#[utoipa::path(
+    context_path = "/v0",
+    security(("JSON web token (JWT) or API key" = [])),
+    responses(
+        (status = 200, description = "All services healthy", body = HealthStatus),
+        (status = 503, description = "One or more services unhealthy", body = HealthStatus)
+    )
+)]
+#[get("/cluster_healthz")]
+pub(crate) async fn get_health(state: WebData<ServerState>) -> impl Responder {
```

Comment:
Why not `Result<HttpResponse, ManagerError>`? Or just `HttpResponse` if no error is thrown.

Reply:
(instead of `impl Responder`)